account_tax_balance/models/account_tax.py

```diff
@@ -3,66 +3,28 @@
 # © 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import models, fields, api
-from openerp.tools.safe_eval import safe_eval
+from openerp import models, fields, api, _
 
 
 class AccountTax(models.Model):
     _inherit = 'account.tax'
 
     balance = fields.Float(
-        string="Total Balance", compute="_compute_balance",
-        search='_search_balance')
+        string="Total Balance", compute="_compute_balance")
```

Comment:
For better readability, instead of removing the comma and put the `)` in the same line, please just remove the `search=...` line and keep the other line. I'm converting also my style habits lately to this one to be more diff-friendly, although it takes more lines:
```
balance = fields.Float(
    string="Total Balance", compute="_compute_balance",
)
```